--- build_android.sh	2022-02-23 21:34:59.417370300 +0100
+++ build_android.sh	2022-02-23 21:34:39.351188000 +0100
@@ -563,7 +563,7 @@
   echo "Adding pathname: `dirname $CXXPATH`"
   # `AndroidBinariesPath` could be used by user-config-*.jam
   export AndroidBinariesPath=`dirname $CXXPATH`
-  export PATH=$AndroidBinariesPath:$PATH
+  export PATH="$AndroidBinariesPath:$PATH"
   export AndroidNDKRoot=$AndroidNDKRoot
   export AndroidTargetVersion32=$ANDROID_TARGET_32
   export AndroidTargetVersion64=$ANDROID_TARGET_64
